python_modules/dagster/dagster/_utils/error.py

```diff
@@ -94,7 +135,17 @@ def serializable_error_info_from_exc_info(
     exc_type = check.not_none(exc_type, additional_message=additional_message)
     e = check.not_none(e, additional_message=additional_message)
     tb = check.not_none(tb, additional_message=additional_message)
-    from dagster._core.errors import DagsterUserCodeProcessError
+
+    from dagster._core.errors import DagsterUserCodeExecutionError, DagsterUserCodeProcessError
+
+    if isinstance(e, DagsterUserCodeExecutionError) and _should_redact_user_code_error():
+        masked_tb_exc = _get_masked_traceback_exception(with_instruction=True)
+        sys.stderr.write(
+            _serializable_error_info_from_tb(
+                _get_masked_traceback_exception(with_instruction=False)
+            ).to_string(),
+        )
+        return _serializable_error_info_from_tb(masked_tb_exc, no_cause=True)
```

Comment:
related to above inline - I think you can simplify this and remove some bool toggling if you just do the raw prefixing [2] here directly

Reply:
Could you elaborate a bit on this? I think this may have been resolved by the most recent commit but I don't know if I fully follow

Reply:
ya you got it